pkg/crc/config/viper_config.go

```diff
@@ -37,16 +37,17 @@ type Config struct {
 	changedConfigs map[string]interface{}
 	// allSettings holds all the config settings
 	allSettings map[string]*Setting
+	configFile  string
 }
 
-func syncViperState(changedConfigs map[string]interface{}, viper *viper.Viper) error {
+func syncViperState(configFile string, changedConfigs map[string]interface{}, viper *viper.Viper) error {
 	encodedConfig, err := json.MarshalIndent(changedConfigs, "", " ")
 	if err != nil {
 		return fmt.Errorf("Error encoding configuration to JSON: %v", err)
 	}
 	err = viper.ReadConfig(bytes.NewBuffer(encodedConfig))
 	if err != nil {
-		return fmt.Errorf("Error reading configuration file '%s': %v", constants.ConfigFile, err)
+		return fmt.Errorf("Error reading configuration file '%s': %v", configFile, err)
```

Comment:
this could be `viper.ConfigFileUsed()`, but I suspect this is gone after a few commits ;)